tests/tests-easy-digital-downloads.php

```diff
@@ -27,7 +27,9 @@ public function test_constants() {
 
 		// Plugin Folder Path
 		$path = str_replace( 'tests/', '', plugin_dir_path( __FILE__ ) );
```

Comment:
It's not the / on the end of tests, its actually the / that ends the path
Example:
..../wp-content/plugins/Easy-Digital-Downloads/
vs 
..../wp-content/plugins/Easy-Digital-Downloads\